web/views/FcRequestsTrack.vue

```diff
@@ -109,21 +236,48 @@ export default {
       });
     },
     actionExport(studyRequests) {
-      // TODO: implement export here
-      console.log('export', studyRequests);
-    },
-    onActionBulk(type, options) {
-      const studyRequests = this.selectedStudyRequests;
-      const actionOptions = options || {};
-      if (type === 'export') {
-        this.actionExport(studyRequests, actionOptions);
+      if (studyRequests.length === 0) {
+        this.setToast(REQUESTS_STUDY_DOWNLOAD_NO_SELECTION);
+        return;
       }
+      const data = studyRequests.map((item) => {
```

Comment:
we may wish to expand this into one row per individual study, complete with requisite lat / long info.   (we can do that when we tackle lat / long for requested studies, though.)

at that point, it may also be worth extracting the CSV data generation into its own function, so that this action can be relatively straightforward (and so that we can test that generation if need be).